src/EditorFeatures/Core/GoToDefinition/IGoToSymbolService.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.Editor.GoToDefinition
+{
+    internal interface IGoToSymbolService : ILanguageService
+    {
+        Task GetDefinitionsAsync(GoToDefinitionContext context);
```

Comment:
this should probably be GoToSymbolContext given that this may be for more than definitions.